boards/beekeeb/piantor/kb.py

```diff
@@ -0,0 +1,52 @@
+import board
+import digitalio
+
+from storage import getmount
+
+from kmk.kmk_keyboard import KMKKeyboard as _KMKKeyboard
+from kmk.scanners.keypad import KeysScanner
+
+# VBus pin config
+VBUS_PIN = board.VBUS_SENSE  # RPi Pico
+# VBUS_PIN = board.GP29      # WeAct RP2040 + resistors on Piantor PCB
```

Comment:
As a person with the WeAct on a Piantor, `board.GP29` results in an error (attribute not found), it appears as `board.A3` instead.

Reply:
Thanks! I've updated the comment to board.A3 instead of board.GP29